src/FSharp.Data.DesignTime/paket.references

```diff
@@ -0,0 +1,8 @@
+FSharp.Core framework: netstandard2.0
+File: ProvidedTypes.fsi .
+File: ProvidedTypes.fs .
```

Comment:
we can also have paket manage the paket-file includes, mapping the particular file into this directory at the specified path (in this case ./ProvidedTypes.fs)